src/EditorFeatures/TestUtilities/TestExperimentationService.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Composition;
+using Microsoft.CodeAnalysis.Experiments;
+using Microsoft.CodeAnalysis.Host.Mef;
+
+namespace Microsoft.CodeAnalysis.Test.Utilities
+{
+    [Shared]
+    [Export(typeof(MockExperimentationService))]
+    [ExportWorkspaceService(typeof(IExperimentationService), WorkspaceKind.Test), PartNotDiscoverable]
+    internal sealed class MockExperimentationService : IExperimentationService
```

Comment:
should the file be renamed to MockExperimentationService to match the type?

Reply:
Renamed the type instead. Good catch